src/app/screens/Accounts/Show/index.tsx

```diff
@@ -143,7 +143,10 @@ function AccountScreen() {
 
     if (nostrPrivateKey === currentPrivateKey) return;
 
-    if (currentPrivateKey && !confirm(t("nostr.private_key.warning"))) {
+    if (
+      currentPrivateKey &&
+      prompt(t("nostr.private_key.warning")) !== "CONFIRM"
```

Comment:
I would allow lower and upercase letters. 

do we know the account name here? is it a good idea to let people type the account name?

Reply:
> I would allow lower and upercase letters.

+1

> is it a good idea to let people type the account name?

Yes, typing the account name sounds like a good option!
(it might also make people change it to their own preferred name after observing that they have the default name!)

Reply:
Done.

Reply:
If the user is typing an incorrect user name, then should we throw some error to let them know?

Reply:
Yes, I think the user should know. Maybe we can add a toast telling the user that the private key hasn't been changed?

Reply:
https://user-images.githubusercontent.com/86483059/220166668-e70eac93-ede3-44ea-872f-731468b3eb53.mp4

Does it looks good?